client/scripts/views/modals/create_community_modal.ts

```diff
@@ -387,13 +571,20 @@ const CreateCommunityModal: m.Component<CreateCommunityAttrs, CreateCommunitySta
           active: vnode.state.activeForm === 'offchain',
           onclick: () => { vnode.state.activeForm = 'offchain'; return null; },
         }),
+        m(TabItem, {
+          label: 'ERC20',
+          active: vnode.state.activeForm === 'erc20',
+          onclick: () => { vnode.state.activeForm = 'erc20'; return null; },
+        }),
         app.user.isSiteAdmin && m(TabItem, {
           label: 'Substrate',
           active: vnode.state.activeForm === 'substrate',
           onclick: () => { vnode.state.activeForm = 'substrate'; return null; },
         }),
       ]),
-      vnode.state.activeForm === 'offchain' ? m(OffchainCommunityForm) : m(SubstrateForm)
+      vnode.state.activeForm === 'offchain' && m(OffchainCommunityForm),
```

Comment:
Can we use an enum here, for all the activeForm options, rather than raw strings?